### PR TITLE
Removing more bundled pytest imports

### PR DESCRIPTION
**Pull request by bsipocz**
and some other minor import cleanup in the same files.

These were sadly missed out in #6729 and only noticed them after that one was merged.

(``grep -r pytest astropy|grep import|grep tests.helper``)

### CODE REVIEW

**Comment by saimn** [comment]
Oups, I missed quite a few ! Thanks for taking care of this :wink:

File: astropy/coordinates/tests/test_api_ape5.py [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
 from numpy import testing as npt
-from ...tests.helper import (pytest, quantity_allclose as allclose,
+from ...tests.helper import (quantity_allclose as allclose,
                              assert_quantity_allclose as assert_allclose)
-raises = pytest.raises
 
 from ... import units as u
 from ... import time
 from ... import coordinates as coords
-from ..errors import *
```
Comment (MSeifert04) [inline astropy/coordinates/tests/test_api_ape5.py]: Either that's a good catch or something very weird. But the tests passed ...

Comment (bsipocz) [reply astropy/coordinates/tests/test_api_ape5.py]: None of those errors/warnings got used in this file, thus the removal.

Comment (MSeifert04) [reply astropy/coordinates/tests/test_api_ape5.py]: In that case: Good catch 👍

**Comment by bsipocz** [comment]
@MSeifert04 - I wasn't aware that there was a convention of import order for 3rd party packages. For std lib ones I try to follow @astrofrog's favourite of ordering them by line-length.

Also, I note that the "cleanup import" part didn't mean to change the order but to remove unused imports!

**Comment by saimn** [comment]
I use alphabetical order for imports, since it's the easiest to do (with a Vi macro) and is - I think - a widely used convention. As many convention it is also much more readable when you get used to it, but I can live with randomly sorted imports :grin:

**Comment by MSeifert04** [comment]
That's the first time I heard of "ordering by line-length" that sounds ... <s>odd</s> fun. 😄

Okay, it was just an inconsistency I found, it shouldn't be a blocker. We just shouldn't do import-reordering in the import-subsections and we should be fine. :)

**Comment by bsipocz** [comment]
OK, I'll push an alphabetical reordeting for 3rd party, so don't push merge just yet.

**Comment by MSeifert04** [comment]
Do you mean putting `import pytest` elsewhere? Because I really wanted to avoid wholesale import reorderings...

**Comment by bsipocz** [comment]
well, in this PR just to put them after numpy. But if you're fine with it as is, I'm happy, too and then can leave to discuss the line length ordering vs lexicographical up to you and @astrofrog ;)

**Comment by MSeifert04** [comment]
I don't care much about the ordering, it just struck me as odd that two subsequent PRs used different locations. That's why I commented.

**Comment by MSeifert04** [comment]
But if we want to discuss this I'm all for increasing ordering according based on the SHA256 checksum of the imported modules source code. That will be fun!

**Comment by bsipocz** [comment]
@MSeifert04 - Too bad that there isn't enough reaction emojis here, the previous, though I quite like the idea; should have deserved a :trollface: 😉

**Comment by MSeifert04** [comment]
I removed the comment again. 😅 I regretted that joke a bit because it sounded as if I were making fun of the "increasing line-length ordering". That wasn't intended so I removed it.

File: astropy/coordinates/tests/test_api_ape5.py [hunk 2]
```diff
@@ -31,6 +31,8 @@
 else:
     HAS_SCIPY = True
 
+raises = pytest.raises
```
Comment (mhvk) [inline astropy/coordinates/tests/test_api_ape5.py]: In every other file we seem to import `raises` from `tests.helper` - maybe do that here too? Alternatively, I'd still prefer `from pytest import raises`.

Comment (bsipocz) [reply astropy/coordinates/tests/test_api_ape5.py]: Indeed, I'll change this to be consistent with the rest, to import from tests.helper.

**Comment by bsipocz** [comment]
At the end I didn't changed the pytest import order given that there were some weak evidence in the modules touched here that they were following the line length ordering ;)
(or was just a few imports not in alphabetical...)

**Comment by mhvk** [comment]
Great, merging!

**Pull request merged**